third_party/dml/DirectMLX.h

```diff
@@ -2644,6 +2671,54 @@ namespace dml
         return output;
     }
 
+    struct BatchNormalizationGradOutputs
+    {
+        Expression outputGradient;
```

Comment:
The convention for these output structs is to omit "output" from the field names. See `RandomGeneratorOutputs`, `MaxPoolingOutputs`, and `GRUOutputs` for example.